app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,18 @@
   </head>
 
   <body>
+    <% if flash[:notice] %>
+    <div class="">
+      <p class="notice"><%= notice %></p>
+    </div>
+  <% end %>
+
+  <% if flash[:alert] %>
+    <div class="">
+      <p class="alert"><%= alert %></p>
+    </div>
+  <% end %>
+
```

Comment:
- [ ] There should be absolutely NO logic in views. Views are only for displaying information that became available to it from its corresponding controller not generating it.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)
I have gone ahead and made some comments on your codebase see affected files below:

**Don't know where to start or how to?** See  [Rails View Helpers Best Practices](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)